backend/drone.js

```diff
@@ -0,0 +1,60 @@
+const dgram = require("dgram");
+const wait = require("waait");
+const app = require("express")();
+const http = require("http").Server(app);
+const io = require("socket.io")(http);
+const throttle = require("lodash/throttle");
+const commandDelays = require("./commandDelays");
+
+const PORT = 8889;
+const HOST = "192.168.10.1";
+const drone = dgram.createSocket("udp4");
+drone.bind(PORT);
+
+function parseState(state) {
+  return state.split(":").map((x) => x.split(":"));
+}
+
+const droneState = dgram.createSocket("udp4");
+droneState.bind(8890);
+
+drone.on("message", (message) => {
+  console.log(`--> : ${message}`);
+
+  // io.sockets.emit('status', message.toString());
+});
+
+function handleError(err) {
+  if (err) {
+    console.log("ERROR");
+    console.log(err);
+  }
+}
+
+const commands = ["command", "battery?", "takeoff", "land", "flip l", "down x"];
+
+const i = 0;
+
+drone.send("command", 0, "command".length, PORT, HOST, handleError);
```

Comment:
what's the purpose of `"command".length`?

Reply:
I believe this whole line might have been for debugging purposes, but has no actual value? 🤔

Reply:
the purpose of the `command.length` is that we need to tell the drone how many characters the command content, when I remove the `command.length` then the drone will not flying...

Reply:
yeah, but it's `"command".length`, which will always be 7

Reply:
I mean `command.length` of each commands that drone have, like (`battery?` have 8, `land` have 4 and so on)
![commands](https://user-images.githubusercontent.com/58105956/109992039-2a234980-7d0b-11eb-9d93-be36d20478ee.jpg)

Reply:
oooh, I understand. thanks for the explanation - one needs to indeed send "command" as the first command and then the actual command itself